src/main/kotlin/pt/up/fe/ni/website/backend/service/ProjectService.kt

```diff
@@ -2,17 +2,24 @@ package pt.up.fe.ni.website.backend.service
 
 import org.springframework.data.repository.findByIdOrNull
 import org.springframework.stereotype.Service
-import pt.up.fe.ni.website.backend.dto.entity.ProjectDto
 import pt.up.fe.ni.website.backend.model.Project
+import pt.up.fe.ni.website.backend.dto.entity.ProjectDto
+import pt.up.fe.ni.website.backend.model.Account
 import pt.up.fe.ni.website.backend.repository.ProjectRepository
 
 @Service
-class ProjectService(private val repository: ProjectRepository) {
+class ProjectService(private val repository: ProjectRepository, private val accountService: AccountService) {
 
     fun getAllProjects(): List<Project> = repository.findAll().toList()
 
     fun createProject(dto: ProjectDto): Project {
         val project = dto.create()
+
+        for (idAccount in dto.teamMembersIds.orEmpty()) {
+            val account = accountService.getAccountById(idAccount)
+            project.teamMembers.add(account)
+        }
```

Comment:
```suggestion
       dto.teamMembersIds?.forEach {
            val account = accountService.getAccountById(it)
            project.teamMembers.add(account)
        }
```
Using forEach avoids the need for `orEmpty()`, the function won't execute if the collection is null